deps/rabbit/src/rabbit.erl

```diff
@@ -300,6 +300,8 @@
 %% 100 ms
 -define(BOOT_STATUS_CHECK_INTERVAL, 100).
 
+-define(DEFAULT_CREDIT, {400, 200}).
```

Comment:
We already have this default defined in https://github.com/rabbitmq/rabbitmq-server/blob/9af2be3e63aff222fa52f3df2bf6a2b1ba89179d/deps/rabbit/Makefile#L84

Defining yet another default in another place (`rabbit.erl`) for the case of user misconfiguration is hard to justify. As @michaelklishin suggested in the discussion, let's make the node booting fail instead by `throw()`ing an error.